languages/elixir/exercises/concept/basic-strings/.docs/introduction.md

```diff
@@ -0,0 +1,46 @@
+Strings in Elixir are delimited by double quotes, and they are encoded in UTF-8:
+
+```elixir
+iex> "Hi! ¡Hola! Cześć! Привет!"
+"Hi! ¡Hola! Cześć! Привет!"
+```
+
+Strings can be concatenated using the `<>/2` operator:
+
+```elixir
+iex> "Welcome to" <> " " <> "New York"
+"Welcome to New York"
+```
+
+Strings in Elixir support interpolation using the `#{}` syntax:
+
+```elixir
+iex> "6 * 7 = #{6 * 7}"
+"6 * 7 = 42"
+```
+
+Elixir provides many functions for working with strings in the _String module_.
+
+```elixir
+iex> String.downcase("HELLO")
+"hello"
+
+iex> String.last("12345")
+"5"
+```
+
+To put a newline character in a string, use the `\n` escape code:
+
+```elixir
+"1\n2\n3\n"
+```
+
+To comfortably work with texts with a lot of newlines, use the triple-double-quote heredoc syntax instead:
+
+```elixir
+"""
+1
+2
+3
+"""
+```
```

Comment:
I am not exactly sure how to integrate `iex` into any of the exercises, since one of the main ways students might interact with elixir for the first time may be through the website since it will offer in-browser coding/testing/analysis not requiring students to have any toolchain at all.  So I have been making examples appear like this:
````
```elixir
string_variable = "Hi! ¡Hola! Cześć! Привет!"
```
````